src/index.html

```diff
@@ -3,11 +3,345 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Thrivetalk</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header class="header">
+      <section class="header__section section">
+        <div class="header__top">
+          <span class="header__logo logo">
+            Thrivetalk
+          </span>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#home" class="nav__link nav__link-active">
+                  Home
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#about" class="nav__link">
+                  About
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#services" class="nav__link">
+                  Services
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#blog" class="nav__link">
+                  Blog
+                </a>
+              </li>
+            </ul>
+          </nav>
+          <a href="#contact" class="header__button-top">
```

Comment:
```suggestion
          <a href="#contact" class="header__button--top">
```
два `-` для модификатора